components/Shared/Icons/index.jsx

```diff
@@ -325,14 +325,13 @@ export const IconGitHub = forwardRef((props, ref) => (
     width='48'
     height='48'
     viewBox='0 0 48 48'
-    fill='none'
     xmlns='http://www.w3.org/2000/svg'
     ref={ref}
     {...props}
   >
     <path
+      fill={props.fill}
```

Comment:
Should we set a `defaultProp` for this? Or does it default to `none` anyway?

Reply:
I think the fill is so context-dependent that adding a default won't make a _huge_ difference...